### ISSUE TITLE
TypeError: needs a 'context' argument.

### ISSUE THREAD

**Issue by sagarrakshe** [rstacruz/mocha-jsdom#3]
I am trying to run a test for a ember application using **mocha-jsdom** and **chai**. (Here is my test file http://pastebin.com/0rE8aai8)

Node version:  **v0.10.35**
Npm verson:  **2.1.17**
Machine:  **OSX Yosemite (v10.10)**

When I run `$ mocha test.js` it gives me the following error:

```
test
 1) "before all" hook


0 passing (1s)
1 failing

1) test "before all" hook:
   TypeError: needs a 'context' argument.
    at new Window (/Users/sagar/zlemma/new-design-zcloud/static/zcloud/node_modules/jsdom/lib/jsdom/browser/Window.js:39:26)
    at Object.exports.jsdom (/Users/sagar/zlemma/new-design-zcloud/static/zcloud/node_modules/jsdom/lib/jsdom.js:46:16)
    at processHTML (/Users/sagar/zlemma/new-design-zcloud/static/zcloud/node_modules/jsdom/lib/jsdom.js:220:24)
    at Object.exports.env.exports.jsdom.env (/Users/sagar/zlemma/new-design-zcloud/static/zcloud/node_modules/jsdom/lib/jsdom.js:122:5)
    at Context.<anonymous> (/Users/sagar/zlemma/new-design-zcloud/static/zcloud/node_modules/mocha-jsdom/index.js:43:22)
    at Hook.Runnable.run (/usr/local/lib/node_modules/mocha/lib/runnable.js:218:15)
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:259:10)
    at Object._onImmediate (/usr/local/lib/node_modules/mocha/lib/runner.js:276:5)
    at processImmediate [as _immediateCallback] (timers.js:354:15)

```

And this works on my colleague's machine (same configuration) but doesn't work on my machine.

**Comment by rstacruz** [comment]
Try deleting node_modules and reinstalling your modules (`npm install`).

**Comment by sagarrakshe** [comment]
Nope. Still getting the same error.
Package json file http://pastebin.com/qVEYE0xE

**Comment by dmwyatt** [comment]
Getting same issue.

**Comment by dmwyatt** [comment]
I edited `mocha-jsdom`'s package.json and replaced `"jsdom": "^1.0.0-pre.7",` with `"jsdom": "^3.1.2",`.

This fixed this issue, but I'm not sure why `mocha-jsdom` was pinned to such an old version of jsdom, so maybe there's side effects that I'm unaware of.

**Comment by sagarrakshe** [comment]
Yes, it worked for me too.

**Comment by rstacruz** [comment]
This is rather strange: mocha-jsdom has its jsdom dependency marked as `*`, so it'll use whatever version is the latest. The version `1.0.0` was a devDependency only to be installed when actually working *on* mocha-jsdom.

Nonetheless, it's been bumped (thanks @nacyon!) and will be available as 0.3.0 pretty soon.
